api/app/tests/test_google_floods_api.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+from app.googleflood import get_google_floods_gauges
+from app.main import app
+from fastapi.testclient import TestClient
+
+client = TestClient(app)
+
+
+@pytest.mark.vcr(
+    match_on=["uri", "method"],
+    filter_headers=["Authorization"],
+    filter_query_parameters=["key"],
+)
+def test_get_google_floods_gauges():
+    """
+    This test is not used in the API, but is used to test the get_google_floods_gauges function
+    It is used to ensure that the get_google_floods_gauges function returns a valid GeoJSON object
+    """
+    gauges = get_google_floods_gauges(["BD"])
+    assert gauges["type"] == "FeatureCollection"
+    for feature in gauges["features"]:
```

Comment:
Note that this would not test anything if length of gauges["features"] is 0

Reply:
Good call, I'll add a length check